Tests/PostgresNIOTests/New/PSQLChannelHandlerTests.swift

```diff
@@ -0,0 +1,183 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 06.01.21.
+//
+
+import XCTest
```

Comment:
Ditch Xcode template headers.
```suggestion
import XCTest
```